common/src/main/scala/net/psforever/types/Vector3.scala

```diff
@@ -326,4 +326,11 @@ object Vector3 {
       z
     )
   }
+
+  def PlanarRotateAroundPoint(point: Vector3, center: Vector3, radians : Float) : Vector3 = {
```

Comment:
So, after realizing that this is z-axis rotation that should be in the same vein as function `Rz`, I double checked and discovered that `Rz` is (probably) written incorrectly when calculating the y-coordinate value.  Shouldn't be fixed, yet, as I'm expecting the wrong behavior as the correct result in some cases.